packages/backend-common/src/config.ts

```diff
@@ -40,7 +40,13 @@ export async function loadBackendConfig(options: {
   const secretEnumerator = await createConfigSecretEnumerator({
     logger: options.logger,
   });
-  const { config } = await newLoadBackendConfig(options);
+  const { config } = await newLoadBackendConfig({
+    argv: options.argv,
+    remote: options.remote,
+    ...(options.additionalConfig
+      ? { additionalConfigs: [options.additionalConfig] }
+      : {}),
+  });
```

Comment:
Hi, is it possible to change `additionalConfig?: AppConfig` to `additionalConfigs?: AppConfig[]` on the line 37 to enable passing **multiple** additional configuration sources when bootstrapping backstage?

Such a change should also make it possible to use the original line if I don't miss anything here.
`const { config } = await newLoadBackendConfig(options);`

Thank you 🙏

Reply:
Not really, that would be a breaking change. I could make it `AppConfig | AppConfig[]` to make it backwards compatible, but not sure that it's worth it - would you have a real use of that at this point or did you mention it just for symmetry's sake?

Reply:
Iirc it's not in mainline yet so fine to break, best double check tho

Reply:
Given that this change has not been released yet (scheduled for Tuesday, 18 April), will this count as a breaking change? I'm sorry for lacking knowledge of your development process, this is my first time contributing to Backstage 👀 

At this point, given that the top-level signature (backend-common) accepts only a single config, it seems like there's no way to pass more than one config anyway to backend-app-api, is there? 

In our team, we have been looking in moving Backstage to a combination of AWS Secrets Manager and AWS Parameter Store as configuration sources hence are interested in having two data sources. If there's no way to do that, we can definitely find a way to survive with one :)

Reply:
Yeah it's been released as part of the `-next` line but not mainline. Alright I allowed it to break now then, @Rugvip check it out again to see if it seems alright!

Reply:
@ilya-kochetkov-cko None of my code is holy, you can delete it for all I care. As long as the end result is the best solution to the problem is all I care about.